packages/loader/container-loader/src/retriableDocumentStorageService.ts

```diff
@@ -99,6 +128,14 @@ export class RetriableDocumentStorageService implements IDocumentStorageService
                 await this.delay(retryAfter);
             }
         } while (!success);
+        if (numRetries > 0) {
+            this.logger.sendTelemetryEvent({
+                eventName: `StorageCallsRetried_${fetchCallName}`,
+                numRetries,
+                totalTime: TelemetryLogger.formatTick(performance.now() - startTime),
```

Comment:
duration here. Also please check what's the name of the property we use for count in getDeltas() - we should use same names across repo - simplifies a lot looking at telemetry (I think it's 'count')

Reply:
It is named as "retry"